.github/workflows/default.yml

```diff
@@ -0,0 +1,33 @@
+name: Polygon Avail Light CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+        # A repository can have up to 10GB of caches.
+        # Check https://github.com/actions/cache for details.
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
```

Comment:
We can actually avoid specifying toolchain here since action can read from rust-toolchain.toml file. Ill update file and remove toolchain spec in workflow.